packages/hardhat-plugin/src/ignition-test-helper.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  Deployer,
+  DeploymentResultSuccess,
+  IgnitionError,
+  IgnitionModuleDefinition,
+  IgnitionModuleResult,
+  MemoryJournal,
+  ModuleParameters,
+} from "@ignored/ignition-core";
+import { Contract } from "ethers";
+import fs from "fs-extra";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { buildAdaptersFrom } from "./buildAdaptersFrom";
+import { buildArtifactResolverFrom } from "./buildArtifactResolverFrom";
+
+export class IgnitionTestHelper {
```

Comment:
Just a clarification, in Hardhat you can't expose a function exclusively to tests, so this should also work for production deployments

Reply:
I think the current implementation makes sense in scripts as well. I will rename the class to `IgnitionHelper` to not confuse by emphasizing one usage.